src/Util/GeometryUtil/GeometryUtil.js

```diff
@@ -0,0 +1,292 @@
+   * @return {ol.geom.Multipoint|ol.geom.MultiPolygon|ol.geom.MultiLinestring} A Multigeometry.
+   */
+  static mergeGeometries(geometries) {
+    const geomType = geometries[0].getType();
+    let mixedGeometryTypes = false;
+    geometries.forEach(geometry => {
+      if (geomType !== geometry.getType()) {
+        mixedGeometryTypes = true;
+      }
+    });
+    if (mixedGeometryTypes) {
+      // Logger.warn('Can not merge mixed geometries into one multigeometry.');
+      return undefined;
+    }
+    let multiGeom;
+    let append;
+    switch (geomType) {
+      case 'Polygon':
+        multiGeom = new OlGeomMultiPolygon();
+        append = multiGeom.appendPolygon.bind(multiGeom);
+        break;
+      case 'Point':
+        multiGeom = new OlGeomMultiPoint();
+        append = multiGeom.appendPoint.bind(multiGeom);
+        break;
+      case 'LineString':
+        multiGeom = new OlGeomMultiLineString();
+        append = multiGeom.appendLineString.bind(multiGeom);
+        break;
+      default:
+        return undefined;
+    }
+    geometries.forEach(geom => append(geom));
+    return multiGeom;
+  }
+
+  /**
+   * Takes two or more polygons and returns a combined polygon.
+   *
+   * @param {ol.geom.Geometry[] | ol.Feature[]} polygons An array of ol.Feature
+   *  or ol.geom.Geomerty instances of type polygon.
+   * @param {String} projection A projection as EPSG code.
+   * @returns {ol.geom.Geometry | ol.Feature} A Feature or Geometry with the
+   * combined area of the polygons.
+   */
+  static union(polygons, projection = 'EPSG:3857') {
+    const geoJsonFormat = new OlFormatGeoJSON({
+      dataProjection: 'EPSG:4326',
+      featureProjection: projection
+    });
+    let invalid = false;
+    const geoJsonsFeatures = polygons.map((geometry) => {
+      const feature = geometry instanceof OlFeature
+        ? geometry
+        : new OlFeature({geometry});
+      if (feature.getGeometry().getType() !== 'Polygon') {
+        invalid = true;
+      }
+      return geoJsonFormat.writeFeatureObject(feature);
+    });
+    if (invalid) {
+      // Logger.warn('Can only create union of polygons.');
+      return undefined;
+    }
+    const unioned = union(...geoJsonsFeatures);
+    const feature = geoJsonFormat.readFeature(unioned);
+    if (polygons[0] instanceof OlFeature) {
+      return feature;
+    } else {
+      return feature.getGeometry();
+    }
+  }
+
+  /**
+   * Finds the difference between two polygons by clipping the second polygon from the first.
+   *
+   * @param {ol.geom.Geometry | ol.Feature} feature An ol.geom.Geoemtry or ol.Feature
+   * @param {ol.geom.Geometry | ol.Feature} feature An ol.geom.Geoemtry or ol.Feature
+   * @param {String} projection A projection as EPSG code.
+   *
+   * @returns {ol.geom.Geometry | ol.Feature} A Feature or Geometry with the area
+   *  of polygon1 excluding the area of polygon2.
+   */
+  static difference(polygon1, polygon2, projection = 'EPSG:3857') {
```

Comment:
It would be possible to pass linestring geometries (e.g.) as well here, wouldn't it? If true, one could be more general here, e.g. ... Find the difference between two geometries ...

Reply:
`turf.js` only supports difference for polygons.